src/raftstore/store/worker/apply.rs

```diff
@@ -39,6 +39,7 @@ use raftstore::store::peer_storage::{self, write_initial_state, write_peer_state
 use raftstore::store::peer::{parse_data_at, check_epoch, Peer};
 use raftstore::store::metrics::*;
 
+const WRITE_BATCH_RECOMMENDED_KEYS: usize = 128;
```

Comment:
As we have benched, when the write batch grow over about a hundred keys, the extra benefits is very trivial, on the other hand we should response the caller of raftkv asap, so we use 128 as the max keys one write batch should contains.